napalm/iosxr/iosxr.py

```diff
@@ -145,7 +145,9 @@ def compare_config(self):
         else:
             return self.device.compare_config().strip()
 
-    def commit_config(self):
+    def commit_config(self, message=''):
+        if not message:
+            raise NotImplementedError('Commit message not implemented for this platform')
```

Comment:
Why not? It is supported by IOSXR and the underlying library: https://github.com/fooelisa/pyiosxr/blob/master/pyIOSXR/iosxr.py#L577

All you have to do is:

```python
         if self.replace:
             self.device.commit_replace_config(comment=message)
         else:
             self.device.commit_config(comment=message)
````

Reply:
Because @dbarrosop and @ktbyers said to do it this way until somebody who knows IOS-XR can add that functionality.

Reply:
I just added it.